### ISSUE TITLE
Unit test word count is inaccurate

### ISSUE THREAD

**Issue by wjjensen** [mischasan/aho-corasick#10]
The word count for acism_t got changed in [3a4021f](https://github.com/mischasan/aho-corasick/commit/3a4021fa2e1cf6ef18240a20e91e42dbf264ffd9), possibly because you are using a different words.gz in your working environment than is checked in to the repo.  This causes unit tests to falsely fail.

I fixed it in my [fork](https://github.com/wjjensen/aho-corasick/commit/c8072b58ec4772f738e58eba201509064af0a287), not sure if you want to either revert the word count, or update words.gz.

**Comment by mischasan** [comment]
Thanks for both fixes. The copy onto the stack was a dubious optimization
that I never measured,ever since using a cranky old 32-bit box where gcc
really needed every general register it could get ... making bp server two
jobs was actually a win !?

**Comment by wjjensen** [comment]
Yeah, I could see how the performance may swing depending on architecture.  I'm always using it in x86_64, but some other targets may be more performant with the original version.

**Comment by mischasan** [comment]
It doesn't apply anywhere but x86 (32) and even there it only applies to
large inputs. So it's getting binned.

**Comment by mischasan** [comment]
The fix is in for a nonmatching suffix error. Apologies for the time it took.
